source/MaterialXFormat/Util.h

```diff
@@ -45,6 +45,14 @@ StringSet loadLibraries(const FilePathVec& libraryFolders,
                         const StringSet& excludeFiles = StringSet(),
                         XmlReadOptions* readOptions = nullptr);
 
+/// Set file name values to resolved values. The resolve invlides apply the Elements resolver, an option relative to absoulte path conversion, and
```

Comment:
Some typos: "The resolve invlides apply the Elements resolver, an option relative to absoulte path conversion"